packages/lib/modules/vebal/vote/Votes/MyVotes/actions/submit/modal/SubmitVotesPreview.tsx

```diff
@@ -204,17 +202,24 @@ export function SubmitVotesPreview({
       </Card>
 
       <HStack alignItems="stretch" spacing="sm" w="full">
-        {false && (
-          <Card flex="1" variant="subSection">
-            <Text variant="special">Potential incentives (1w)</Text>
-            <HStack spacing="xs">
-              <Text fontSize="lg" fontWeight={700} variant="special">
-                {toCurrency(optimizedRewardValue, { abbreviated: false })}
-              </Text>
-              <MyIncentivesAprTooltip totalWithVotesOptimized={totalWithVotesOptimized} />
-            </HStack>
-          </Card>
-        )}
+        <Card flex="1" variant="subSection">
+          <Text variant="special">Potential incentives (1w)</Text>
+          <HStack spacing="xs">
+            <Text fontSize="lg" fontWeight={700} variant="special">
+              {totalInfo.totalRewardValue !== undefined ? (
+                toCurrency(totalInfo.totalRewardValue, { abbreviated: false })
+              ) : (
+                <>&mdash;</>
```

Comment:
Cannot we return '-'?

Reply:
from `toCurrency`? This is used in multiple places and I suppose that some of those don't necessarily have to return a dash when no value passed.